src/fieldModifierLocation/field.ts

```diff
@@ -24,7 +24,7 @@ function separateResolvedData(
             resolvedData = value;
             unResolvedData =
                 field.schema.multiple === true
-                    ? value.map((file: any) => file.uid)
+                    ? value.map(({ file }: any) => file.uid)
```

Comment:
@abhishek305  would be better if we can have a proper type instead of any

Reply:
Considering the use here we can add a generic object for now @MaitriGurey1